config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml

```diff
@@ -1069,3 +1069,64 @@ periodics:
       - --test_args=--ginkgo.focus=\[sig-network\] --ginkgo.skip=\[Slow\]|\[Flaky\]|\[Feature:Networking-IPv6\]|\[Disruptive\]|\[Feature:ServiceLoadBalancer\]|\[Feature:PerformanceDNS\] --minStartupPods=8
       - --timeout=120m
       image: gcr.io/k8s-testimages/kubekins-e2e:v20180730-8b7ab3104-master
+
+- interval: 2h
+  name: ci-kubernetes-e2e-containerd-gce-calico-serial
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+    preset-kubernetes-e2e-containerd: "true"
+    preset-kubernetes-e2e-enable-calico: "true"
+  spec:
+    containers:
+    - args:
+      - "--timeout=340"
+      - --root=/go/src
+      - --repo=github.com/containerd/cri=master
+      - "--repo=k8s.io/examples=master"
+      - --scenario=kubernetes_e2e
+      - --
+      - --check-leaked-resources
+      - --env=KUBE_MASTER_EXTRA_METADATA=user-data=/go/src/github.com/containerd/cri/test/e2e/master.yaml,containerd-configure-sh=/go/src/github.com/containerd/cri/cluster/gce/configure.sh,containerd-env=/workspace/test-infra/jobs/e2e_node/containerd/containerd-master/env
+      - --env=KUBE_NODE_EXTRA_METADATA=user-data=/go/src/github.com/containerd/cri/test/e2e/node.yaml,containerd-configure-sh=/go/src/github.com/containerd/cri/cluster/gce/configure.sh,containerd-env=/workspace/test-infra/jobs/e2e_node/containerd/containerd-master/env
+      - --extract=gs://koonwah-repo-pub/devel/v1.11.2-beta.0.20+21777efd99bd59
+      - --gcp-node-image=gci
+      - --gcp-nodes=4
+      - --gcp-zone=asia-southeast1-a
+      - --ginkgo-parallel=1
+      - --provider=gce
+      - --test_args=--ginkgo.focus=[Disruptive\]||\[Feature:PerformanceDNS\] --minStartupPods=8
+      - --timeout=120m
```

Comment:
This appears to have two, wildly different timeout values?

Reply:
thank you for point it out, I have updated the timeout value.